parlai/nn/lr_scheduler.py

```diff
@@ -91,6 +92,7 @@ def load_state(self, states):
         if states.get('warmup_scheduler') and getattr(self, 'warmup_scheduler', None):
             self.warmup_scheduler.load_state_dict(states['warmup_scheduler'])
         self._number_training_updates = states.get('number_training_updates', 0)
+        self.step(self._number_training_updates)
```

Comment:
what is this change?

Reply:
oh boy, it's a bit of a pytorch hack.

Until we do a step(), the LR from a FRESHLY loaded model may report an incorrect value. The changes in train model that force a reload surfaced this.